test/unit/foreman/renderer/renderers_shared_tests.rb

```diff
@@ -137,5 +137,15 @@ module RenderersSharedTests
         renderer.render(source, @scope)
       end
     end
+
+    test "should load hosts" do
```

Comment:
I'd move these tests to a dedicated scope class for report templates.

Reply:
same as the previous one